README.md

```diff
@@ -2,4 +2,23 @@
 
 Repository for sharing Knative resources to ArtifactHub.
 
-https://artifacthub.io/docs/topics/repositories/knative-client-plugins/
\ No newline at end of file
+https://artifacthub.io/docs/topics/repositories/knative-client-plugins/
+
+### Adding new artifact metadata
+
+The metadata stored in `plugins` directory are periodically indexed by ArtifactHub to list our plugins per `kn-client-plugin` type of AH.
```

Comment:
Would expand AH, though relatively clear that this means "Artifact Hub" (or even skip it)
```suggestion
The metadata stored in `plugins` directory is periodically indexed by Artifact Hub to list our plugins per `kn-client-plugin` type.
```